core/src/scss/components/main-nav/_main-nav--light.scss

```diff
@@ -1,189 +1,233 @@
 @charset "UTF-8";
 
-.su-main-nav {
-  &.su-main-nav--light {
+.su-main-nav--light {
+  $_color-map: $main-nav-light-colors;
```

Comment:
Please namespace this variable.

Reply:
Changed from $_color-map to $_su-main-nav-color-map (combining both @sherakama and @JBCSU 's suggestions)